draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -416,25 +416,27 @@ oracle {{XWING}}.
 
 This section instantiates three concrete KEMs:
 
-1. `QSF-SHA3-256-ML-KEM-768-P-256` {{qsf-p256}}: A hybrid KEM using the QSF
-   combiner based on ML-KEM-768 and P-256.
-2. `KitchenSink-HKDF-SHA-256-ML-KEM-768-X25519` {{ks-x25519}}: A hybrid KEM
-   using the KitchenSink combiner based on ML-KEM-768 and X25519.
-3. `QSF-SHA3-256-ML-KEM-1024-P-384` {{qsf-p384}}: A hybrid KEM using the QSF
-   combiner based on ML-KEM-1024 and P-384.
+1. `QSF-KEM(ML-KEM-768,P-256)-XOF(SHAKE256)-KDF(SHA3-256)` {{qsf-p256}}:
+   A hybrid KEM using the QSF combiner with SHA3-256 as the hash function
+   based on ML-KEM-768 and P-256, along with SHAKE256 as the key derivation XOF.
+2. `KitchenSink-KEM(ML-KEM-768,X25519)-XOF(SHAKE256)-KDF(HKDF-SHA-256)` {{ks-x25519}}:
```

Comment:
```suggestion
2. `KitchenSink-KEM(ML-KEM-768,X25519)-XOF(SHAKE256)-KDF(HKDF-SHA-256)` {{ks-x25519}}:
```

Hm I worry about a comma in a label like this, have we done this in other i*tf crypto protocols? Is it fine in practice?

Reply:
merging for now, but may want to keep an eye on it